src/main/java/frc/robot/commands/FindAndGotoNote.java

```diff
@@ -81,7 +96,7 @@ public void execute() {
             // and the pitch to the target is greater than we expect
             if (!(RobotState.getInstance().isAutonomous() && target.getPitch() > 17 &&
                     ((PoseEstimator.getInstance().getPose().getX() > 6 && !FieldState.getInstance().onRedAlliance()) ||
-                            (PoseEstimator.getInstance().getPose().getX() < 10 && FieldState.getInstance().onRedAlliance())))) {
+                            (PoseEstimator.getInstance().getPose().getX() < 10 && FieldState.getInstance().onRedAlliance()))) || DriverStation.isTeleop()) {
```

Comment:
I don't understand this logic... are you assuming that if it's greater than you expect, then it must be a note in a place other than the one you were after?

Reply:
This basically checks of we're past a certain X position. we must be going after a center note, and then if the pitch of the target is too high, we assume it's past the center line, and therefore don't want to get it.